jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/editor/provider/SAPVersionDependenciesManager.java

```diff
@@ -20,20 +20,24 @@
 import org.fusesource.ide.sap.ui.Activator;
 
 public class SAPVersionDependenciesManager implements IDependenciesManager {
-	
-	static final String SAP_VERSION_621 = "6.2.1.redhat-084";
-	static final String SAP_VERSION_630 = "6.3.0.redhat-187";
-	public static final String LAST_SAP_VERSION = SAP_VERSION_630;
+
+	/* Fuse 6.2.1.R7 */
+	static final String SAP_VERSION_621_LATEST = "6.2.1.redhat-183";
+	/* Fuse 6.3.0.R2 */
+	static final String SAP_VERSION_630_LATEST = "6.3.0.redhat-254";
+
+	public static final String LAST_SAP_VERSION = SAP_VERSION_630_LATEST;
+
 	private static Map<String, String> camelToSAPVersionMapping;
 
 	static {
 		camelToSAPVersionMapping = new HashMap<>();
-		camelToSAPVersionMapping.put("2.15.1", SAP_VERSION_621);
-		camelToSAPVersionMapping.put("2.17.0", LAST_SAP_VERSION);
-		camelToSAPVersionMapping.put("2.17.3", LAST_SAP_VERSION);
-		// TODO: update me with every new release of camel and sap supported in tooling
+		camelToSAPVersionMapping.put("2.15.1", SAP_VERSION_621_LATEST);
+		camelToSAPVersionMapping.put("2.17.0", SAP_VERSION_630_LATEST);
+		camelToSAPVersionMapping.put("2.17.3", SAP_VERSION_630_LATEST);
+		// TODO: update me with every new release of camel and sap supported in

```

Comment:
The latest version is returned only on two cases
  - there is a version without 'redhat' such as '2.17.1'
  - the given camel version is unknown such '2.18.1.redhat-000015'

Otherwise, it will parse the 'redhat' version and returns the appropriated camel-sap version.

So, the question is whether we want to use such general logic or the exact mapping. Since we are in a process of decoupling the camel version I would prefer the general logic.